doc/source/build_hints.rst

```diff
@@ -1803,11 +1803,70 @@ The following options are available to select a subset of drivers:
     Independently of options that control global behavior, drivers can be individually
     enabled or disabled with those options.
 
+    .. note::
+
+        <driver_name> above and below is *generally*, but not systematically the short driver name.
+
+        Some drivers may also be grouped together for build purposes.
+
+        - A number of "raw" raster drivers (ACE2, BT, BYN, CPG, CTable2, DIPX, DOQ1,
+          DOQ2, EHDR, EIR, ENVI, FAST, GenBIN, GSC, GTX, MFF2, ISCE, KRO, MFF, LAN,
+          LCP, LOSLAS, NDF, NTv2, PAUX, PNM, ROIPAC, RRASTER, SNODAS) are controlled
+          by the GDAL_ENABLE_DRIVER_RAW option.
+
+        - Planetary raster formats (PDS, PDS4, ISIS, ISIS3, VICAR) are controlled by
+          the GDAL_ENABLE_DRIVER_PDS option.
+
+        - The ECW and JP2ECW raster drivers are controlled by the GDAL_ENABLE_DRIVER_ECW option.
+
+        - The GSAG, GSBG and GS7BG raster drivers are controlled by the GDAL_ENABLE_DRIVER_GSG option.
+
+        - The MrSID and JP2MrSID raster drivers are controlled by the GDAL_ENABLE_DRIVER_MRSID option.
+
+        - The NITF, RPFTOC and ECRGTOC raster drivers are controlled by the GDAL_ENABLE_DRIVER_NITF option.
+
+        - The Interlis 1 and Interlis 2 vector drivers are controlled by the GDAL_ENABLE_DRIVER_ILI option.
+
+        - The WFS and OAPIF vector drivers are controlled by the GDAL_ENABLE_DRIVER_WFS option.
+
+        - The AVCBIN and AVCE00 vector drivers are controlled by the GDAL_ENABLE_DRIVER_AVC option.
+
+        - The DWG and DGNv8 vector drivers are controlled by the GDAL_ENABLE_DRIVER_DWG option.
+
+        There might be variations in naming, e.g. :
+
+        - the "AIGRID" raster driver is controlled by GDAL_ENABLE_DRIVER_AIG.
```

Comment:
Should this be AAIGrid (two A's)?

Reply:
nope, I meant https://gdal.org/drivers/raster/aig.html, but I actually see this follows the regular naming pattern, so no need to mention it